syntax = "proto3";

message TableId {
    uint64 id = 1;
}